docs/guides/node-operator-manual.md

```diff
@@ -181,6 +181,13 @@ You would need an RPC endpoint - a local node / RPC provider (eg Alchemy/Infura)
 
 ### Submitting the keys
 
+> Please note, that the withdrawal address should be added to the Lido Node Operators Registry before it can submit the signing keys. Adding an address to the Node Operators Registry happens via DAO voting. When providing withdrawal address to be added to the Node Operators Registry, keep in mind the following:
+> - it is the address that will receive rewards;
+> - it is the address you will be using for key management;
```

Comment:
let's directly say "submitting keys to Lido"?

Reply:
Fixed.
Missed out on this comment, sorry. I suggest merging this pr now and reviewing the doc once again before the upcoming testnet session.